pkg/services/ldap/ldap.go

```diff
@@ -261,6 +261,7 @@ func (server *Server) getSearchRequest(
 	return &ldap.SearchRequest{
 		BaseDN:       base,
 		Scope:        ldap.ScopeWholeSubtree,
+		SizeLimit:    1000,
```

Comment:
Is there any point in making this change without also implementing support for splitting up search request?

Reply:
As I mentioned in the description - no, not really. But at least it would explicitly works for 1000 users